packages/server/postgres/Dockerfile

```diff
@@ -2,9 +2,14 @@ FROM postgres:15.4
 
 ADD extensions /extensions
 
-RUN apt-get update && apt-get install -y build-essential locales
+RUN apt-get update && apt-get install -y \
```

Comment:
+1 I found adding --no-install-recommends shaves off some seconds from the install process. I like using v15 moving forward though!

https://github.com/ParabolInc/parabol/blob/ec5d15bd851430cc0856a1455ee8c741ec40b26a/packages/server/postgres/Dockerfile#L5

Reply:
no `--no-install-recommends`?
perfectly fine if not, should only affect first install